Assets/Mirage/Runtime/CharacterSpawner.cs

```diff
@@ -100,6 +91,15 @@ private void OnServerStarted()
         /// <param name="sceneOperation">The type of scene load that happened.</param>
         public virtual void OnClientFinishedSceneChange(string scenePath, SceneOperation sceneOperation)
         {
+            if (ClientObjectManager != null)
+            {
+                ClientObjectManager.RegisterPrefab(PlayerPrefab);
```

Comment:
I think this would register the prefab multiple times, Might be better to register it once when the client starts

Reply:
It currently registers everytime scene changes. Mirage itself calls register prefabs internally in clientobjectmanager. This is why i had to move this here. So we either need to change how mirage registers ro once in clientobjectmanager or we need to do this. I trace it and every time player bit join then disconnect and join it called register spawn in that class